hoodie-common/src/main/java/com/uber/hoodie/common/model/FileSlice.java

```diff
@@ -73,17 +79,26 @@ public String getBaseInstantTime() {
     return baseInstantTime;
   }
 
+  public String getPartitionPath() {
+    return partitionPath;
+  }
+
   public String getFileId() {
     return fileId;
   }
 
+  public HoodieFileGroupId getFileGroupId() {
```

Comment:
This is a little weird to me because of the following : A FileGroup is a collection of a List<FileSlice>, now a filegroupId lies in a file slice. Do we plan to add the HoodieFileGroupId in both HoodieFileGroup as well as FileSlice ?

Reply:
Yes, I have now added HoodieFileGroupId in both file-slice and file-group